src/commands/core/asset/create.ts

```
@@ -204,11 +204,11 @@ export default class AssetCreate extends TransactionCommand<typeof AssetCreate>
     return jsonUri
   }
 
-  private formatAssetResult(result: any, explorer: ExplorerType): string {
+  private async formatAssetResult(result: any, explorer: ExplorerType): Promise<string> {
     return `--------------------------------
   Asset: ${result.asset}
   Signature: ${base58.deserialize(result.signature! as Uint8Array)[0]}
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)